### PR TITLE
progress bar animation

### PR DESCRIPTION
**Pull request by AShivangi**
**xbmgmt program --test_mode**

Comment out code in "SubCmdProgram.cpp" to try out some other use cases

### CODE REVIEW

File: src/runtime_src/core/tools/common/ProgressBar.h [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+// Include files
+// Please keep these to the bare minimum
+#include <string>
+#include <chrono>
+
+// ------ N A M E S P A C E ---------------------------------------------------
+
+// Temporary color/ cursor objects until the supporting color library becomes available
+namespace ec {
```
Comment (rozumx) [inline src/runtime_src/core/tools/common/ProgressBar.h]: Please move this functionality to its own separate header file.

File: src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp [hunk 2]
```diff
@@ -18,6 +18,8 @@
 // Local - Include Files
 #include "SubCmdProgram.h"
 #include "tools/common/XBUtilities.h"
+#include "tools/common/XBHelpMenus.h" //to be removed later. Currently imports: is_esc_enabled()
```
Comment (rozumx) [inline src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp]: Lets move is_esc_enable() to the XBUtilities.

**Comment by AShivangi** [comment]
reset this please

**Comment by AShivangi** [comment]
retest this please

**Pull request merged**